src/client/hooks/useKeyEvent/index.tsx

```diff
@@ -7,13 +7,16 @@ import { blockTypes } from '/client/constants/tetriminos';
 import { keyboard } from '/client/constants/keyboard';
 import Block from '/client/models/block';
 
-export const useKeyEvent = ({ setIsPause, setMatrix, setBlock }
+export const useKeyEvent = ({ setIsPause, setMatrix, setBlock, isLeader }
   : {
     setIsPause: Dispatch<SetStateAction<boolean>>,
     setMatrix: Dispatch<SetStateAction<Matrix>>,
     setBlock: Dispatch<SetStateAction<Block>>,
+    isLeader?: boolean,
   }): void => {
+  console.log('Leader 1:', isLeader); // ok
   useKey((_key: number, { keyCode }: { keyCode: number }) => {
+    console.log('Leader 2:', isLeader); // TODO: prop is not visible here, how to pass it?
```

Comment:
```suggestion
    setCurrentUser((currentUser) => console.log('Leader 2:', currentUser.isLeader)); // TODO: prop is not visible here, how to pass it?
```

need to be inside setCurrentUser